packages/website/docs/_components_pages/fiori/SideNavigation/SideNavigation.mdx

```diff
@@ -26,3 +27,8 @@ Quick action items allow for providing access to frequent functionality.
 SideNavigation is enabled to be used inside a responsive popover. This example shows how to achieve overlay mode with correct design and interaction.
 
 <OverlayMode />
+
+### Defining Custom Width
+The width property sets the horizontal size of the side navigation panel. The panel must be at least 16 rem wide to maintain readability and layout stability. You can set a larger width as needed, but it cannot be reduced below 16 rem.
```

Comment:
Configuring the width is done with css, not a property so the usage of the word 'property' in the first sentence might be confusing. Consider rephrasing it, e.g. "The horizontal size of the side navigation panel can be customized through CSS."